src/screens/Login/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Text } from 'react-native-paper';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import { useAppDispatch } from 'src/hooks';
+import { useAppSelector } from 'src/hooks';
+
+const LoginScreen = () => {
+  const dispatch = useAppDispatch();
+  const isAuthorized = useAppSelector(state => state.auth.isAuthorized);
```

Comment:
We will show the login screen only for unathorized users, the screen doesn't need to handle it, so remove this